runtime/oti/j9.h

```diff
@@ -322,13 +322,15 @@ static const struct { \
 	(J9_ARE_NO_BITS_SET((romFieldShape)->modifiers, J9AccVolatile) || (J9CLASS_UNPADDED_INSTANCE_SIZE(fieldClazz) <= sizeof(U_64))))
 #define J9_VALUETYPE_FLATTENED_SIZE(clazz) (J9CLASS_HAS_4BYTE_PREPADDING((clazz)) ? ((clazz)->totalInstanceSize - sizeof(U_32)) : (clazz)->totalInstanceSize)
 #define IS_REF_OR_VAL_SIGNATURE(firstChar) ('L' == (firstChar) || 'Q' == (firstChar))
+#define IS_VAL_SIGNATURE(firstChar) ('Q' == (firstChar))
```

Comment:
@tajila I thought it would simplify the code to add this. Lmk if this is good.

Reply:
one suggestion is to rename it to `IS_QTYPE`